utils/zerovec/derive/examples/derives.rs

```diff
@@ -4,10 +4,20 @@
 
 use zerovec::ule::AsULE;
 use zerovec::ule::EncodeAsVarULE;
+use zerovec::ule::VarULE;
+use zerovec::ule::ULE;
 use zerovec::*;
 
+fn validate_foo_ule(ule: &FooULE) -> Result<(), ZeroVecError> {
+    if ule.a == 0 {
+        return Err(ZeroVecError::parse::<Foo>());
+    }
+    Ok(())
+}
+
 #[repr(packed)]
-#[derive(ule::ULE, Copy, Clone)]
+#[derive(ule::ULE, Copy, Clone, Debug)]
+#[ule(validate_with = "validate_foo_ule")]
```

Comment:
nit: this should be `zerovec::ule`. we do that consistently elsewhere.

It's also fine for it to be `zerovec::validate_with`, or `zerovec(validate_with =)` though that last pattern isn't used so far in this crate